EMIP/1-1000/EMIP00816.xml

```diff
@@ -29,34 +29,226 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                   <collection>EMIP</collection>
                   <idno facs="EMIP/Codices/816/">Chelekot Silasie 75</idno>
                </msIdentifier>
+
                <msContents>
-                  <summary>Homilies on Right Faith, ድርሳነ፡ ርቱዓ፡ ሃይማኖት።; Homily on Wälätä Maryam by Hǝrǝyaqos, ድርሳነ፡ ወለተ፡ ማርያም፡</summary>
-                  <msItem xml:id="ms_i1">
-                     <textLang mainLang="gez">Gǝ‘ǝz</textLang>
-                  </msItem>
+                  <summary>Homilies on Right Faith, ድርሳነ፡ ርቱዓ፡ ሃይማኖት።; Homily on Walatta Māryām by Hǝrǝyāqos, ድርሳነ፡ ወለተ፡ ማርያም፡</summary>
+
+<msItem xml:id="ms_i1">
+<locus from="1r" to="136r" facs="001"/>
+<title xml:lang="gez" type="complete" ref="LIT2222RetuaH">Rǝtuʿa Hāymānot</title>
+<textLang mainLang="gez"/>
+
+<msItem xml:id="ms_i1.1">
+<locus from="1r" to="6v" facs="001"/>
+<title xml:lang="gez" type="complete" ref="LIT2222RetuaH#Introduction"/>
+<incipit xml:lang="gez">ድርሳን፡ ዘደረሰ፡ አባ፡ ሰላማ፡ በእንተ፡ ሥላሴ፡ ወሥጋዌሁ፡ ለወልደ፡ እግዚአብሔር፡ ሕያው፡ ወሕማማቲሁ፡ ወዕርገቱ፡ ውስተ፡ ሰማያት፡</incipit>
+<note>The attribution of the introductory homily to ʾabbā <persName ref="PRS8387SalamaI">Salāmā</persName> is unique among the Rǝtuʿa Hāymānot manuscripts.</note>
+<textLang mainLang="gez"/>
```

Comment:
I don't think that you need to add the @textLang statement as you have specified the language in the @title statement

Reply:
Daria Elagina with whom I trained stressed that it needs to be repeated both in <title> and as <textLang>, I may delete of course, or request some explanation from "the superiors"?

Reply:
lang in title refers to the language in which the title is written (in this particular description). Might be English as well. Two different things. In this particular case, title does not need lang since the information is retrieved from another file.

Reply:
Thank you Daria, sorry for bothering you. I'm deleting xml:lang from title

Reply:
That's helpful for me, thanks for this brief discussion